modules/tinymce/src/themes/silver/main/ts/Theme.ts

```diff
@@ -22,7 +24,41 @@ const registerOptions = (editor: Editor) => {
 export default (): void => {
   ThemeManager.add('silver', (editor): Theme => {
     registerOptions(editor);
-    const { dialogs, popups, renderUI }: RenderInfo = Render.setup(editor);
+
+    // When using the ui_of_tomorrow, the popup sink is placed as a sibling to the
+    // editor, which means that it might be subject to any scrolling environments
+    // that the editor has. Therefore, we want to make the popup sink have an overall
+    // bounds that is depeendent on its scrolling environment. We don't know that ahead
+    // of time, so we use a cell whose value will change if there is a scrolling context.
+    const popupSinkBounds = Cell<() => Boxes.Bounds>(() => Boxes.win());
```

Comment:
If this is just local mutation maybe use the `let` keyword instead?

Reply:
- [ ] use let with location mutation instead of cells